notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<blockquote>These feature vectors are directly fed into VITS without the need for conversion to a text-based intermediate representation.</blockquote><p>could you please add link on VITS?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added